spacy/pipeline/transition_parser.pyx

```diff
@@ -642,7 +647,7 @@ class Parser(TrainablePipe):
                     raise ValueError(Errors.E149) from None
         return self
 
-    def _init_batch(self, teacher_step_model, docs, max_length):
+    def _init_batch(self, teacher_pipe, docs, max_length):
```

Comment:
Given that this method is only used by the `distill` method, could we have a more appropriate name like `_init_batch_from_teacher`?